src/providers/fileManifestState.tsx

```diff
@@ -51,6 +51,7 @@ export type FileManifestState = {
   isSummaryLoading: boolean;
   requestParams?: URLSearchParams;
   requestURL?: string;
+  setOfFormFacetNames: Set<FileFacet["name"]>;
```

Comment:
What about renaming this to help indicate these are the selected set of form facet names (e.g. `selectedFormFacetNames`)?